packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/DatagridExamples.2/Datagrid.scss

```diff
@@ -0,0 +1,8 @@
+@import './Datagrid';
```

Comment:
Lets drop the DataGrid import and add the styles directory or import both sass files into the example and class bind as needed